src/components/button/styles.js

```diff
@@ -0,0 +1,36 @@
+import styled, { css } from 'styled-components';
+import { Link } from 'react-router-dom';
+
+const sizes = {
+  small: 'width:92px;height:36px;',
```

Comment:
You shouldn't hardcode the width. In a real app it's always possible that a string changes. Often you have translations that have different length. The width here can be achieved by setting the padding.